blask-6/app/routes/blog.py

```diff
@@ -0,0 +1,33 @@
+from flask import Blueprint, render_template, redirect, url_for, request
+#last 3 imports are for the new form
+from app.models.blog import BlogPost, BlogPostForm
+
+# Create the blog Blueprint.  Note that all routes on the blog blueprint have
+# '/blog' prepended to them.
+blog = Blueprint('blog', __name__, url_prefix='/blog')
+
+@blog.route('/')  # Accessible at /blog/
+def blog_page():
+    """The blog page."""
+    #objects() method comes from the Flask mongoengine
```

Comment:
Consider `# objects() is a Flask-Mongoengine method that returns all of the BlogPost objects.`